src/ui/states/state-needs-payment-info.svelte

```diff
@@ -67,8 +67,13 @@
     });
 
     if (result.error) {
-      // payment failed, notify user
       processing = false;
+
+      if (result.error.type === "validation_error") {
```

Comment:
`validation_error`s does not affect the setup_intent state so it's safe to keep using the same Stripe Elements instance to complete the purchase